tests/Browser/Pages/Materials/UpdateMaterialTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Tests\Browser\Pages\Materials;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\DB;
+use App\User;
+use App\Category;
+use App\Material;
+use App\Supplier;
+use Faker\Factory as Faker;
+
+class UpdateMaterialTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+     
+    /**
+     * Test link UpdateMaterial.
+     *
+     * @return void
+     */
+    public function testUpdateFood()
```

Comment:
Please make correct function name. I don't see update action in this function. Only check access to update page